src/shared-components/dropdown/style.js

```diff
@@ -0,0 +1,144 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import {
+  COLORS,
+  BOX_SHADOWS,
+  SPACING,
+  ANIMATION,
+  TYPOGRAPHY_CONSTANTS,
+} from '../../constants';
+
+const optionsContainerMaxHeight = '250px';
+
+export const DropdownContainer = styled.div`
+  position: relative;
+  width: 100%;
+  text-align: ${({ textAlign }) => textAlign};
+`;
+
+export const dropdownInputStyle = ({ textAlign }) => css`
+  appearance: none;
+  box-shadow: ${BOX_SHADOWS.clickable};
+  background: ${COLORS.white};
+  background-image: none;
+
+  width: 100%;
+  min-height: ${SPACING.xlarge};
+  max-height: ${SPACING.xlarge};
+
+  border: 1px solid ${COLORS.border};
+  border-radius: 0;
+
+  color: ${COLORS.primary};
```

Comment:
@youngmanalive `bodyText` should always be `purple80`, just wanted to make you aware. This might be a legacy thing that's still in here but just wanted to make sure this is corrected.

Reply:
Should I always be using `purple80` instead of `primary` for body/contenty stuff? 

`primary` is used everywhere across the site and the only usage of `purple80` I see is in `immersiveModal`. Name of `primary` seems much more semantic than `purple80` for all body text. Maybe this is all backlogged todos/legacy stuff, just wanted to clarify that this is the correct route to go.

Lemme know if there's some designy stuff we should go over.